### DISCUSSION TITLE
Trying to make it work on Redmi note 8

### DISCUSSION THREAD

**Discussion by Barranqueiro** [AppGround-io/bluetooth-keyboard-and-mouse-support#236] [Bugs & Problems]
Hi, I was trying to use the app on a Redmi note 8 using a APK file, i configured all, connect to the device I want to use (a Samsung tablet) but i can't make it work (tested with the same tablet using a Motorola one fusion+ as keyboard and worked fine). Is there some connection problem on Xiaomi devices?

**Comment by AppGround-io** [comment]
The Redmi Note 8 only supports the compatibility connection mode. This mode does not work with all devices. You could install [LineageOS](https://lineageos.org/) on your device which supports the normal connection mode that is much more reliable. Then it should be possible to connect to your Samsung tablet.
